### PR TITLE
Standings List View 

### PR DESCRIPTION
**Pull request by MCurran16**
This PR contains the markup and styles for the standings list view. I created a knockout template for it.

### CODE REVIEW

File: assets/js/templates/views/league-details/league-details.jst.jade [hunk 1]
```diff
@@ -1,3 +1,17 @@
+script(type='text/html' id="standings-list-view")
+	tr.table__row
+		td.list-view-content(data-bind="text: $data.rank, \
+				style: {backgroundColor: $data.team_color}")
+		td.list-view-content
+			span.content-text(data-bind="text: $data.team_name")
```
Comment (hunterbrowning) [inline assets/js/templates/views/league-details/league-details.jst.jade]: Good use of a span!

File: assets/js/vm/views/league-details/league-details.js.coffee [hunk 2]
```diff
@@ -45,10 +46,111 @@ do ->
...
 				Content Header Links
          ----------------------------------###
+         @selectedDetailItem = ko.observable("standings")
+
+         ###-----------------------------------
+            Standings Content
+         ----------------------------------###
+         @standingsArray = ko.observableArray([])
+
+         @populateData()
```
Comment (hunterbrowning) [inline assets/js/vm/views/league-details/league-details.js.coffee]: Usually a good idea to have this as the very last call in your constructor as it is temporary.

File: assets/js/templates/views/league-details/league-details.jst.jade (hunk 1 above)
Comment (MCurran16) [inline assets/js/templates/views/league-details/league-details.jst.jade]: gracias

File: assets/js/vm/views/league-details/league-details.js.coffee (hunk 2 above)
Comment (MCurran16) [inline assets/js/vm/views/league-details/league-details.js.coffee]: fixed

**Comment by hunterbrowning** [comment]
LGTM

**Pull request merged**